dask_ml/model_selection/_hyperband.py

```diff
@@ -106,6 +107,32 @@ class HyperbandSearchCV(BaseIncrementalSearchCV):
         optimal. ``aggressiveness=4`` has higher confidence that is likely
         suitable for initial exploration.
 
+    explore : bool, int, default=False
+        If ``explore=True`` is specified, run a custom exploratory search aimed
```

Comment:
Can you think of a 1-sentance explanation of the purpose of `explore`? Even after reading the description I'm still unsure.

Reply:
Thanks, done. This PR still isn't ready for merge; I have a couple todo's to complete in https://github.com/dask/dask-ml/pull/532#issue-297450726.